index.html

```diff
@@ -63,8 +63,447 @@ <h2 class="who-am-i">I’m a software developer</h2>
             
          
 
-        </section> 
+        </section>         
+        <div class="collapse-button">
+            <img src="asset/images/Disabled.png" alt="collapse project-button">
+        </div>          
+
+        <h2 class="works-heading"> MY RECENT WORKS</h2>            
+
+        <section class="place-holder page-1">
+            
```

Comment:
- [ ] Also The main container of the projects has a `display:grid`. Please add it.
- [ ] The section has the proper background image (decorations are always backgrounded images, not `img` tags)

Reply:
display:grid has been added to the main container